server/handlers/eventLogs.js

```diff
@@ -24,11 +25,27 @@ let handlers = {
     if (!isNaN(reqSkip) && reqSkip) {
       skip = reqSkip
     }
+
     c.crn.logs
       .find({ type: { $in: events } }, { sort: [['date', 'desc']] })
       .toArray((err, logs) => {
         if (err) return next(err)
-        res.send(logs)
+
+        const userPromises = logs.map(log => {
+          return new Promise(resolve => {
+            scitran.getUser(log.user, (err, response) => {
```

Comment:
This works but do we really want to make an api call for every log? Is the goal to find the ORCID entries in the logs and replace those with an actual email? If so, should we only make this api call for the logs that have an ORCID? Seems like over time this could get expensive but perhaps it is not a big deal.

Reply:
yea, you're totally right on this one. will update with conditional re: ORCID users

Reply:
I filed #322 as a backlog issue to improve this further in the future.

Reply:
So you think this is ok as is then?

Reply:
Yeah, I think it is cleaner to do it this way and fix the performance consideration with #322 since Google users technically have an email field as well (and it could be different in the future).

Reply:
OK sounds good.